source/getting-started/use_cases/security_analytics.rst

```diff
@@ -22,21 +22,12 @@ Below you can see a list of these components:
 - **System inventory** runs regular scans, collecting inventory data including network interfaces, operating system version, installed applications, running processes, and a list of open ports. 
 - **Security configuration assessment (SCA)** provides a continuous evaluation based on the Center of Internet Security (CIS) benchmarks.
 
-Security analytics tools give users the visibility needed to perform thread prevention actions. The :ref:`FIM <file_integrity>` module is a good example because it detects changes that could compromise data and creates alerts based on them. Then, the agent can be configured to trigger an active response in reply to these alerts.
-
-In addition, these tools assist users in detecting the possible presence of rootkits and collecting inventory data from different sources such as operating systems, processes, applications, ports, and network interfaces. FIM results can be observed in the **Integrity monitoring** module dashboard of the :ref:`Wazuh Kibana plugin<kibana_app>`.
-
-.. thumbnail:: ../../images/getting_started/integrity_monitoring_1.png 
-  :title: FIM overview
-  :align: center
-  :width: 100%
-
-.. thumbnail:: ../../images/getting_started/integrity_monitoring_2.png 
-  :title: FIM dashboard
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-security-analytics.png
```

Comment:
Why does the screenshot of this dashboard look cut at the end?
Was this a mistake? 

![image](https://user-images.githubusercontent.com/79285586/141323730-fedb690c-a524-4af6-a67a-6bd0f9ab5e50.png)